docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -0,0 +1,737 @@
+    after setting all other parameters to connect via a bus.
+
+Creating a User Defaults File
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The first step in creating a defaults file is finding the parameters that work best for your modules. Once you have a set of parameters 
+(across the general, tuning, and advanced tabs) that work well, you can continue. 
+
+1. With your module connected, navigate to the general tab and find the MODULE PRE-SETTINGS
+2. Find the Export button on the right side
+3. Click Export, and you will see a File Explorer window open
+4. Here, choose a name and location for your generated defaults file. In this example I will save the file as “example_defaults.json”
+5. You will then be given the option to import the file directly into the Control Center. In order to use this file to configure other modules, you must import it into the Control Center
+
+.. image:: ../_static/control_center_pics/import_defaults_popup.png
+
+6. After clicking Yes, you will see your imported defaults as an option in the dropdown. This file will now always be available in the Control Center
+7. Once in the Control Center, you can use the Set button exactly as you would with a Vertiq supplied Defaults file
+
+Importing a User Defaults File
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+It is also possible to import externally generated Defaults files into the Control Center. To do so:
+
+#. Select the Import option
+#. Use the File Explorer to find the Defaults file you would like to import
+#. You will see the file appear in the dropdown of other Defaults files, and it is ready for use
+
+******************************************
+Testing Modules with the Control Center
+******************************************
+The Control Center provides a basic suite of testing options to ensure that your module is functioning properly. These options are available in the testing tab, 
+available on the left side.
+
+.. note::
+    Before using the Control Center's testing functionality, we recommend that you increase your module's timeout parameter to 1.5s. 
+    This parameter is available through the tuning tab. If you do not increase the timeout value, your module may fail to spin as expected.
+
+.. warning::
+    Please remove all propellers before interacting with any testing parameters. Failure to do so is dangerous.
+
+In order to set a testing parameter, simply enter a value into the spin box, and hit the down arrow set button. Parameters such as Coast do not have a spin box, 
+and only require that you press the set button to trigger the behavior.
+
+The testing tab for a Veritq 23-06 2200Kv looks as follows:
+
+.. image:: ../_static/control_center_pics/23_testing_tab.png
+    :scale: 50%
+
+Brake
+=========
+This commands the module to brake, which shorts the motor phases, which slows the motor down by dissipating energy in the motor.
+
+.. warning::
```

Comment:
This may make sense to move up to the testing introduction section instead of just living with the brake stuff. Its not really brake specific